app/samples/sample-browsemap-details-window/src/main/java/com/sygic/samples/BrowseMapDetailsWindowActivity.kt

```diff
@@ -17,17 +18,25 @@ class BrowseMapDetailsWindowActivity : CommonSampleActivity() {
         val browseMapFragment = supportFragmentManager.findFragmentById(R.id.browseMapFragment) as BrowseMapFragment
         browseMapFragment.addMapMarkers(
             listOf(
-                MapMarker(48.143489, 17.150560),
-                MapMarker(48.165561, 17.139550),
-                MapMarker(48.155028, 17.155674),
-                MapMarker(48.141797, 17.097001),
-                MapMarker.Builder()
-                    .coordinates(48.162805, 17.101621)
-                    .title("My Marker 1")
+                MapMarker.from(48.143489, 17.150560).build(),
```

Comment:
V tomto souhlasím se Zdenčím, proč by se měl MapMarker jako jediný ViewObject vytvářet pomocí "from", moc se mi to nelíbí :(